docs/products/redis/concepts/high-availability-redis.rst

```diff
@@ -5,44 +5,62 @@ Aiven for Redis®* is available on a variety of plans, offering different levels
+      - Automatic backups to a remote location.
+      - 3 days
     * - **Premium**
-      - Three-node (primary + standby + standby) with top availability characteristics
-      - 30-day
+      - Three-node (primary + standby + standby)
+      - Enhanced high availability with automatic failover among multiple standby nodes if the primary fails.
+      - Automatic backups to a remote location.
+      - 13 days
     * - **Custom**
-      - Four-node (primary + N x standby) with top availability characteristics
-      - 30-day
+      - Custom configurations
+      - Custom high availability and failover features based on user requirements.
+      - Custom backup features based on user requirements.
+      - Custom based on user requirements
 
 
+
+Check out our `Plans & Pricing <https://aiven.io/pricing?product=redis>`_ page for more information. 
+
 Failure handling
 ----------------
 
-- **Minor failures**, such as service process crashes or temporary loss of network access are handled by Aiven automatically in all plans without any major changes to the service deployment. The service automatically restores normal operation once the crashed process is automatically restarted or when the network access is restored.
-- **Severe failure**, such as losing a node entirely in case of hardware or severe software problems, requires more drastic recovery measures. Losing an entire node (virtual machine) could happen, for example, due to hardware failure or a severe enough software failure. The Aiven monitoring infrastructure automatically detects a failing node both when the node starts reporting that its own self-diagnostics is having problems or when the node stops communicating entirely. The monitoring infrastructure automatically schedules a new replacement node to be created.
+- **Minor failures**: Aiven automatically handles minor failures, such as service process crashes or temporary loss of network access, without any significant changes to the service deployment. In all plans, the service instantly restores normal operation by automatically restarting the crashed process or restoring the network access once available.
+- **Severe failure**: In case of severe hardware or software problems, such as losing an entire node, it requires more drastic recovery measures. An entire node (virtual machine) may be lost due to hardware failure or a severe software issue. Aiven's monitoring infrastructure automatically detects a failing node when it reports problems with its self-diagnostics or stops communicating altogether. The monitoring infrastructure then schedules the creation of a new replacement node.
+
 
 .. Note::
-        In case of database failover, the **Service URI** of your service remains the same, only the IP address changes to point to the new master node.
+        In case of database failover, your service's **Service URI** remains the same—only the IP address changes to point to the new master node.
```

Comment:
```suggestion
        In case of database failover, your service's **Service URI** remains the same—only the IP address changes to point to the new primary node.
```